src/mcts/search.cc

```diff
@@ -1139,6 +1139,11 @@ void SearchWorker::FetchSingleNodeResult(NodeToProcess* node_to_process,
   if(node_to_process->depth % 2 == 1)
     penalty = -penalty;
   node_to_process->v = -computation_->GetQVal(idx_in_computation) + penalty;
+  if(node_to_process->v < -1.0) {
+    node_to_process->v = -0.9999;
```

Comment:
If we go with simple clipping like this, I think it should be:
 `if (v < -0.9999) v = -0.9999;... etc`
and maybe should use the min/max style the code uses elsewhere:
`v = std::max(-0.9999, std::min(0.9999, v))`

Reply:
Also this code should only execute if the penalty is enabled.

Reply:
I made those changes as you suggested.